LICENCE.html

```diff
@@ -0,0 +1,236 @@
+<!DOCTYPE html>
+<html lang="en">
+<!-- See http://creativecommons.org/licenses/by-sa/4.0/legalcode -->
+<!-- Human-readable summary: http://creativecommons.org/licenses/by-sa/4.0/ --> 
+<head>
+  <title>Creative Commons — Attribution-ShareAlike 4.0 International — CC BY-SA 4.0</title>
```

Comment:
Hi Colin,
The licence would only apply to the content on gpii.github.io, not to the repositories referenced in the content. I know that we don't use copyleft licences for code, but I thought CC for a document like this one wouldn't restrict the reuse of the code referenced here.
I can remove the ShareAlike to make it less restrictive. 
Should I wait a few days to give others time to comment?